modules/nvidia_plugin/src/cuda_config.hpp

```diff
@@ -4,7 +4,7 @@
 
 #pragma once
 
-#include <cuda/cuda_config.hpp>
+#include <nvidia/nvidia_config.hpp>
```

Comment:
should not we align with the rest of OpenVINO? See https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/openvino/runtime/intel_gpu/properties.hpp and also create a new header for properties with new API?

Reply:
Lets probably close it as adaptation to CUDAPlugin renaming to NVIDIAPlugin and then I will do this adaptation, because it could require change some additional tests and maybe add additional tests ?